frontend/src/v5/ui/components/viewer/cards/cardContext.component.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
```

Comment:
Is there maybe a better term instead of view? I'm just wondering if it's going to get confusing with the "Views" window that already exists. You might end up having views in Views

Reply:
unless you have another suggestion I think I prefer cardView